<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>DOCUMENTS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAccueil">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Accueil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>361</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Qualité</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Hygiène</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>131</width>
     <height>296</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="Button_11_Doc">
      <property name="text">
       <string>Button_11_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_12_Doc">
      <property name="text">
       <string>Button_12_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_13_Doc">
      <property name="text">
       <string>Button_13_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_14_Doc">
      <property name="text">
       <string>Button_14_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_15_Doc">
      <property name="text">
       <string>Button_15_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_16_Doc">
      <property name="text">
       <string>Button_16_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_17_Doc">
      <property name="text">
       <string>Button_17_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_18_Doc">
      <property name="text">
       <string>Button_18_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_19_Doc">
      <property name="text">
       <string>Button_19_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_20_Doc">
      <property name="text">
       <string>Button_20_Doc</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>131</width>
     <height>296</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="Button_1_Doc">
      <property name="text">
       <string>Etude HACCP</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_2_Doc">
      <property name="text">
       <string>Etude Food FRAUD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_3_Doc">
      <property name="text">
       <string>Etude Food Defense</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_4_Doc">
      <property name="text">
       <string>Button_14_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_5_Doc">
      <property name="text">
       <string>Button_5_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_6_Doc">
      <property name="text">
       <string>Button_6_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_7_Doc">
      <property name="text">
       <string>Button_7_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_8_Doc">
      <property name="text">
       <string>Button_8_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_9_Doc">
      <property name="text">
       <string>Button_9_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_10_Doc">
      <property name="text">
       <string>Rapport Qualité</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>230</y>
     <width>131</width>
     <height>296</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="Button_21_Doc">
      <property name="text">
       <string>Button_21_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_22_Doc">
      <property name="text">
       <string>Button_22_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_23_Doc">
      <property name="text">
       <string>Button_23_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_24_Doc">
      <property name="text">
       <string>Button_24_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_25_Doc">
      <property name="text">
       <string>Button_25_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_26_Doc">
      <property name="text">
       <string>Button_26_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_27_Doc">
      <property name="text">
       <string>Button_27_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_28_Doc">
      <property name="text">
       <string>Button_28_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_29_Doc">
      <property name="text">
       <string>Button_29_Doc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Button_30_Doc">
      <property name="text">
       <string>Button_30_Doc</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_44">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Niveau de restriction</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>30</width>
     <height>296</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QPushButton" name="pushButton_1">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_10">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>30</width>
     <height>296</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QPushButton" name="pushButton_11">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_12">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_13">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_14">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_15">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_16">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_17">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_18">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_19">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_20">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>230</y>
     <width>30</width>
     <height>296</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QPushButton" name="pushButton_21">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_22">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_23">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_24">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_25">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_26">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_27">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_28">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_29">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_30">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>crayon.png</normaloff>crayon.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
